geogson/build.gradle

```diff
@@ -0,0 +1,15 @@
+apply plugin: 'java'
+apply plugin: 'me.tatarka.retrolambda'
+
+dependencies {
+    def dependencies = rootProject.ext.dependencies
+
+    compile fileTree(dir: 'libs', include: ['*.jar'])
+
+    compile 'com.google.guava:guava:19.0'
```

Comment:
As far as I know, the guava is a huge library.
Is there anyway we can only use it at compile-time or something like that?